akka-docs/src/main/paradox/stream/operators/Source/unfoldResourceAsync.md

```diff
@@ -14,8 +14,36 @@ Wrap any resource that can be opened, queried for next element (in a blocking wa
 
 ## Description
 
-Wrap any resource that can be opened, queried for next element (in a blocking way) and closed using three distinct functions into a source.
-Functions return @scala[`Future`] @java[`CompletionStage`] to achieve asynchronous processing
+Wrap any resource that can be opened, queried for next element and closed in an asynchronous way with three distinct functions into a source. This operator is the equivalent of @ref[unfoldResource](unfoldResource.md) but for resources with asynchronous APIs.
```

Comment:
@ennru link already in here

Reply:
I meant it might be useful to copy this section from the `unfoldResource` docs:

> Note that there are pre-built unfoldResource-like operators to wrap java.io.InputStreams in Additional Sink and Source converters, Iterator in fromIterator and File IO in File IO Sinks and Sources. Additional prebuilt technology specific connectors can also be found in the Alpakka project.

Reply:
Ah, I see. I agree!

Reply:
:+1: